cfgov/v1/handlers/blocks/feedback.py

```diff
@@ -80,7 +80,20 @@ def get_response(self, form):
                 )
             except (ValueError, TypeError):
                 pass
-            feedback.referrer = self.request.POST.get('referrer', '')
+
+            # The referrer is set manually here instead of being set on the
+            # ModelForm. Whereas MySQL silently truncates varchar fields that
+            # exceed the column length, Postgres does not and will raise a
+            # "value too long for type character varying" error if an
+            # excessively long referrer is saved.
+            #
+            # This code grabs the referrer from the request and truncates it
+            # to at most the length of the feedback model referrer field.
+            referrer = self.request.POST.get('referrer', '')
+            referrer_field = feedback._meta.get_field('referrer')
```

Comment:
I wouldn't have thought to grab the max length directly from _meta. Nice.

Reply:
Thanks @higs4281. I didn't want to have it hardcoded in two places.